src/v0/destinations/fb_custom_audience/recordTransform.js

```diff
@@ -235,6 +214,66 @@ async function processRecordInputs(groupedRecordInputs) {
   return finalResponse;
 }
 
+function processRecordInputsV1(groupedRecordInputs) {
+  const { destination } = groupedRecordInputs[0];
+  const { message } = groupedRecordInputs[0];
+  const { isHashRequired, disableFormat, type, subType, isRaw, audienceId, userSchema } =
+    destination.Config;
+
+  let operationAudienceId = audienceId;
+  let updatedUserSchema = userSchema;
+  if (isEventSentByVDMV1Flow(groupedRecordInputs[0])) {
```

Comment:
Is this `isEventSentByVDMV1Flow` not a redundant check as `processRecordInputs` already handling the code segregation ?

Reply:
processRecordInputsV1 this method is common for EventStream and rETL records v1

Reply:
got it - but ideally whenever we are opening up `record` spec to eventstream, it's better to do on record v2 rather than v1

Reply:
actually it is not much different for Event streams as there no context

Reply:
we can do that too once this has become standard